stories/About/Developer Guide.mdx

```diff
@@ -0,0 +1,236 @@
+import { Meta } from "@storybook/blocks";
+import Image from "next/image";
+
+import supabase from '../assets/supabase_gridIron.png';
+import server from '../assets/supabase_server.png';
+import client from '../assets/client_component.png';
+  
+<Meta title="About/Developer Guide" />
+
+<div className="sb-container">
+  <div className='sb-section-title'>
+    <h1>Developer Guide</h1>
+  </div>
+ <section>
+<h3>Table of Contents</h3>
+      <ol>
+        <li><a href = "#started">Getting Started</a></li>
+        <li><a href = "#push">Pushing local branch to remote branch</a></li>
+        <li><a href = "#connect">Connecting to Supabase</a></li>
+        <li><a href = "#query">Query Supabase data from Next.js</a></li>
+      </ol>
+
+<h3 id = "started">Getting Started</h3>
+<ol>
+  <li>Clone the develop branch to your local machine:</li>
+  <li><code>git clone --single-branch --branch develop https://github.com/LetsGetTechnical/gridiron-survivor.git</code></li>
```

Comment:
This is where the link should go to a new page. Anything outside of the Storybook infrastructure.